xui/additional_files_unzipped/s3_get_versions/s3_get_versions/cb_plugin_218_OrDfUdw_vHnOPmv_SneuueF.py

```diff
@@ -89,6 +89,5 @@ def inbound_web_hook_post(*args, parameters={}, **kwargs):
     except Exception as e:
         error_message = e.args[0]
         # log error message with user 
-        logger.error("User %s failed to get versions of %s" % (user, key))
-        logger.error(error_message)
+        logger.exception("User %s failed to get versions of %s" % (user, key))
         return {"status": False, "message": error_message}
```

Comment:
All these cases of error handling you're doing in these IWHs are a great pattern of showing a basic error msg to the end user and printing the TB to the logs. All this error handling will pay off when customers hit errors (which they will, very frequently).